src/main/scala/net/snowflake/spark/snowflake/Parameters.scala

```diff
@@ -165,7 +169,9 @@ object Parameters {
     PARAM_USE_COPY_UNLOAD -> "true",
     PARAM_USE_PROXY -> "false",
     PARAM_EXPECTED_PARTITION_COUNT -> "1000",
-    PARAM_MAX_RETRY_COUNT -> "10"
+    PARAM_MAX_RETRY_COUNT -> "10",
+    // 2020-02-13 Default authenticator
+    PARAM_AUTHENTICATOR -> "snowflake"
   )
```

Comment:
When defining a new parameters, add separate func to retrieve the parameters. e.g. sfUser() and using that func instead of using parameter name.

Reply:
removed the default value for PARAM_AUTHENTICATOR, updated version checkedin